internal/donor/donorpage/identity_details_test.go

```diff
@@ -94,77 +171,178 @@ func TestPostIdentityDetails(t *testing.T) {
 	existingDob := date.New("1", "2", "3")
 	identityDob := date.New("4", "5", "6")
 
+	identityAddress := place.Address{Line1: "different"}
+
+	// cannot change - yes
+	// cannot change - no
+	// address not matched - yes
+	// address not matched - yes, continue with mismatched
+	// address not matched - no
+	// address not matched - no, continue with mismatched
+	// detail not matched - yes
+	// detail not matched - yes, address not matched
+	// detail not matched - no
+	// detail not matched - no, address not matched
```

Comment:
Do we want to keep these or rely on test names below?

Reply:
👁️ I missed deleting my notes